common/src/main/java/com/wynntils/functions/CharacterFunctions.java

```diff
@@ -151,6 +151,20 @@ public List<Class<? extends Model>> getModelDependencies() {
         }
     }
 
+    public static class ManaPctFunction extends DependantFunction<Float> {
+        @Override
+        public Float getValue(String argument) {
+            int currentMana = ActionBarModel.getCurrentMana();
+            int maxMana = ActionBarModel.getMaxMana();
+            return ((float) currentMana / maxMana * 100.0f);
+        }
```

Comment:
Perhaps inline `currentMana` and `maxMana`, as 
```java
        @Override
        public Float getValue(String argument) {
            return ((float) ActionBarModel.getCurrentMana() / ActionBarModel.getMaxMana() * 100.0f);
        }
```

Reply:
Okay either way imo

Reply:
Although, shouldn't you round the number to 1 or 2 decimals?

Reply:
This returns a Float. The rounding is done in presentation?

Reply:
> This returns a Float. The rounding is done in presentation?

FunctionManager handles it. Right